modules/dynamic-operator/Makefile

```diff
@@ -105,15 +112,10 @@ run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go ${ARGS}
 
 .PHONY: docker-build
-docker-build: export CGO_ENABLED=0
-docker-build: export GOOS=linux
-docker-build: export GOARCH=amd64
-docker-build: export GO111MODULE=on
 docker-build: export LDFLAGS="-X main.ver=$(VERSION)"
-docker-build: export COMPOSE_LDFLAGS="-X main.ver=$(VERSION)"
 docker-build: test ## Build docker image with the manager.
-	go build -a -o bin/manager -ldflags=$(LDFLAGS) main.go
-	cd plugins/docker-compose && go build -a -o ../../bin/docker-compose-plugin -ldflags=$(COMPOSE_LDFLAGS) . && cd -
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o bin/manager -ldflags=$(LDFLAGS) main.go
+	cd plugins/docker-compose && CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o ../../bin/docker-compose-plugin -ldflags=$(COMPOSE_LDFLAGS) . && cd -
```

Comment:
I think you forgot about `LDFLAGS` and `COMPOSE_LDFLAGS` here

Reply:
no, they are passed as arguments to `go build`.
Still, I reverted back everything except goos and goarch that are platform specific